### ISSUE TITLE
issue while tyring to access Open road form  in a desktop application

### ISSUE THREAD

**Issue by snithya2208** [microsoft/WinAppDriver#1417]
I am trying to implement automation for a desktop application which has been developed using .net and OpenRoad..I could access the .net windows using winapp driver but when it comes to OpenRoad form it couldn't access the elements in the OpenRoad form..I wool like to know if WinAppDriver  supports openRroad forms?

**Comment by anunay1** [comment]
are the elements shown in inspect.exe, if not you cannot use windows application driver.

**Comment by liljohnak** [comment]
@snithya2208 What is OpenRoad?

**Comment by snithya2208** [comment]
@anunay1 yes I could see the name of the element using visualUI verify .. I have used the same to locate the element defining it as a Windows’s element .. the application is developed with a combination of .net and OpenRoad .. I could navigate through the win forms without any issue , at a particular step it opens a openroad form and my test got struck there as it couldn’t locate the element

**Comment by snithya2208** [comment]
> @snithya2208 What is OpenRoad?

It is a development technology similar to .net I believe .. my issue is the software opens up two different applications (one is the .net and the other is OpenRoad).. I will have to switch between these two applications for testing a workflow
